docs/source/contrib.rst

```diff
@@ -32,9 +32,25 @@ Visit the `Kornia face detection demo on the Hugging Face Spaces
 
 Image Segmentation
 ------------------
-
 .. autofunction:: connected_components
 
+
+Segment Anything (SAM)
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. autoclass:: kornia.contrib.sam.architecture.Sam
+    :members:
+    :undoc-members:
+
+.. autoclass:: kornia.contrib.sam.SamPredictor
+    :members:
+    :special-members: __call__,
+
+.. autoclass:: kornia.contrib.sam.model.SamPrediction
+    :members:
```

Comment:
Add also the result datastructure

Reply:
What do you mean? is it just add the `SegmentationResults`?